commit 1562fe8b627427d0e35fc7919f9cd4a76bf64f72 (refs/changes/60/2960/1)
Author: Richard Kiene <richard.kiene@joyent.com>
Date:   2017-11-17T00:28:43+00:00 (1 year, 11 months ago)
    
    joyent/manta-thoth#156 64bit node binaries are not debuggable by mdb when a 32bit node binary is in PATH
    Reviewed by: Bryan Cantrill <bryan@joyent.com>
    Approved by: Bryan Cantrill <bryan@joyent.com>

diff --git a/bin/thoth b/bin/thoth
index 2ac3141..3e3e5cc 100755
--- a/bin/thoth
+++ b/bin/thoth
@@ -1777,7 +1777,13 @@ handlers.debug = function (client, argv)
 		if (analyzer) {
 			cmd += analyzerCmd(args);
 		} else {
-			cmd += 'mdb $MANTA_INPUT_FILE';
+			/*
+			 * LIBPROC_INCORE_ELF=1 prevents us from loading
+			 * whatever node binary happens to be in $PATH. This is
+			 * necessary when debugging cores that do not match the
+			 * bitness of the node binary found in $PATH.
+			 */
+			cmd += 'LIBPROC_INCORE_ELF=1 mdb $MANTA_INPUT_FILE';
 		}
 
 		setAnalyzer(client, cmd, function (asset) {
diff --git a/package.json b/package.json
index 54eb376..3c6c2c2 100644
--- a/package.json
+++ b/package.json
@@ -1,8 +1,13 @@
 {
 	"name": "manta-thoth",
 	"description": "Manta-based system for core and crash dump analysis",
+<<<<<<< HEAD
+	"version": "1.1.0",
+        "license": "MIT",
+=======
 	"version": "1.0.0",
     "license": "MIT",
+>>>>>>> 3f0c0a2... joyent/manta-thoth#154 Need license info in package.json
 	"author": "Joyent (joyent.com)",
 	"dependencies": {
 		"manta": "^4.0.0",
